bin/lambda-cfn.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+var cli = require('../lib/cli');
+var d3 = require('d3-queue');
+
+var parsed;
+try { parsed = cli.parse(process.argv.slice(2), process.env); }
+catch (err) { return finished(err); }
+
+// if (parsed.options.all) {
+  // call a helper function to compile the list of
+  // stacks for the command to loop over
+  //var q = d3.queue(1);
+  //for (var stack in parsed.options.stacks) {
+    // modify parsed and pass to command loop
+    // q.defer(cli.main, modified_parsed)
+    // q.await(function(err,data) {});
+//  }
+// } else {
+  cli.main(parsed, finished);
+
+  function finished(err, data) {
+    if (err) process.stderr.write(err.message + '\n');
```

Comment:
Similar to https://github.com/mapbox/lambda-cfn/pull/51/files#r132270052 we could probably change this line to `if (err) console.error(err)` but I haven't tested this yet. It would avoid for the need to set up error objects like in https://github.com/mapbox/lambda-cfn/pull/51/files#diff-f8849ae9f0614ba9464fdd14c2d98dc0R172. Although this still may or may not be desirable. I assume this was also a convention inherited from cfn-config.